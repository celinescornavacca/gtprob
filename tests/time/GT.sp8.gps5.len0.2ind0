((((B0,(B1,(B2,(B3,B4)))),((C0,(C1,C2)),(C3,C4))),(((A1,A2),(A3,A4)),A0)),((((G0,G1),(G2,(G3,G4))),(((H3,H4),H2),(H0,H1))),((D0,(D1,(D2,(D3,D4)))),((E0,((E3,E4),(E1,E2))),(F0,((F3,F4),(F1,F2)))))));
