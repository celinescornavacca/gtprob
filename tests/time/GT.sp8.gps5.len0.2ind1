((((A0,(A3,A4)),(A1,A2)),(((B1,B2),((B3,B4),B0)),(((C3,C4),C0),(C1,C2)))),((((((D3,D4),D2),D1),D0),((G0,((G1,G2),(G3,G4))),((H3,H4),((H1,H2),H0)))),((E0,((E1,E2),(E3,E4))),((F3,F4),((F1,F2),F0)))));
