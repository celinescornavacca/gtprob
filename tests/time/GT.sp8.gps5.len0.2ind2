(((A0,((A3,A4),(A1,A2))),((B1,(B2,(B3,B4))),B0)),((((D0,D1),(D2,(D3,D4))),(((G0,(G1,G2)),(G3,G4)),((H0,(H3,H4)),(H1,H2)))),((((C3,C4),C2),(C0,C1)),(((E1,E2),((E3,E4),E0)),((F1,F2),((F3,F4),F0))))));
