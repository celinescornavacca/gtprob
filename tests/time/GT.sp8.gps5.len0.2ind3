(((A1,(A2,(A3,A4))),A0),(((((B1,B2),(B3,B4)),B0),(C0,(C1,((C3,C4),C2)))),(((E0,((E3,E4),(E1,E2))),(((F1,F2),F0),(F3,F4))),((((G2,(G3,G4)),(G0,G1)),((H0,(H3,H4)),(H1,H2))),(D0,(((D3,D4),D2),D1))))));
