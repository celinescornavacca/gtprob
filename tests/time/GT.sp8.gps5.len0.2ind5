(((((E2,(E3,E4)),(E0,E1)),((F1,((F3,F4),F2)),F0)),((G0,((G2,(G3,G4)),G1)),((H1,((H3,H4),H2)),H0))),(((A0,(A1,(A2,(A3,A4)))),((B0,B1),((B3,B4),B2))),((C0,((C1,C2),(C3,C4))),((D3,D4),((D1,D2),D0)))));
