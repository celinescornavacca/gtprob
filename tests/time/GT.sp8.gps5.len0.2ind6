((((E0,(E1,(E2,(E3,E4)))),(F0,(((F3,F4),F2),F1))),(((C0,((C3,C4),(C1,C2))),((D0,D1),((D3,D4),D2))),((B3,B4),(B0,(B1,B2))))),(((G0,(G1,((G3,G4),G2))),((H1,(H2,(H3,H4))),H0)),(A0,(A1,(A2,(A3,A4))))));
