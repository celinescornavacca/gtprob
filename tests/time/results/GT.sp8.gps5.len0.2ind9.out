-72.8564056703
